crategen/cli.py

```diff
@@ -1,30 +1,39 @@
-import click
+"""CLI module for converting TES and WES data to WRROC."""
+
 import json
+
+import click
+
 from crategen.converter_manager import ConverterManager
 
+
 @click.command()
-@click.option('--input', prompt='Input file', help='Path to the input JSON file.')
-@click.option('--output', prompt='Output file', help='Path to the output JSON file.')
-@click.option('--conversion-type', prompt='Conversion type', type=click.Choice(['tes-to-wrroc', 'wes-to-wrroc']), help='Type of conversion to perform.')
+@click.option("--input", prompt="Input file", help="Path to the input JSON file.")
+@click.option("--output", prompt="Output file", help="Path to the output JSON file.")
+@click.option(
+    "--conversion-type",
+    prompt="Conversion type",
+    type=click.Choice(["tes-to-wrroc", "wes-to-wrroc"]),
+    help="Type of conversion to perform.",
+)
 def cli(input, output, conversion_type):
```

Comment:
Okay/better to fix only _existing_ typing issues here. But adding the type hints to function/method signatures across the entire codebase in a future PR (ideally very soon, maybe the next PR?).